geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/tests/BasicDistributedTest.java

```diff
@@ -154,16 +156,18 @@ public void testRemoteInvokeAsync() throws Exception {
     String name = getUniqueName();
     String value = "Hello";
 
-    AsyncInvocation async1 = vm0.invokeAsync(() -> remoteBind(name, value)).join().checkException();
-    AsyncInvocation async2 =
-        vm0.invokeAsync(() -> remoteValidateBind(name, value)).join().checkException();
+    AsyncInvocation<Void> async1 =
+        vm0.<Void>invokeAsync(() -> remoteBind(name, value)).join().checkException();
+    AsyncInvocation<Void> async2 =
+        vm0.<Void>invokeAsync(() -> remoteValidateBind(name, value)).join().checkException();
```

Comment:
Note: `join().checkException()` can and should always be replaced with `await()` instead.

I'm not requesting changes... just a comment so you know.

Reply:
Thanks, I fixed it anyway. Might as well clean up while I'm in the area.